microsoft-edge/devtools-guide-chromium/workspaces/index.md

```diff
@@ -56,236 +52,459 @@ The tutorial steps below walk you through this environment setup.
 
-1. Refresh the page.  The heading in the rendered page changes to "I Love Cake", because that string was saved in `index.html` in your mapped `app` directory:
+1. Press **Ctrl+S** (Windows, Linux) or **Command+S** (macOS).
 
-   ![Change HTML from the Sources tool](./index-images/workspaces-workspaces-demo-sources-page-h1.png)
+   Your change is saved, and the asterisk disappears.
 
-1. Open the copy of `index.html` that's in your `app` directory in a text editor, such as Visual Studio Code.
+   You made the change in the file listing, rather than in the DOM tree in the **Elements** tool.
 
-   The `<h1>` element contains the new text, because you made the change using the **Sources** tool's editor to edit `index.html` and then saved the change, and that file was mapped in a workspace (the **Filesystem** tab), indicated by a green dot on the file's icon.
+1. Refresh the page.  The heading in the rendered page changes to "I Love Cake", because that string was saved in `index.html` in your mapped `/Demos/workspace/` directory:
+
+   ![Change HTML from the Sources tool](./index-images/sources-page-h1.png)
+
+1. In the **Sources** tool, right-click `index.html`, and then select **Open in containing folder**.
+
+   File Explorer or Finder opens.
+
+1. Open the copy of `index.html` that's in your `/Demos/workspace/` directory in a text editor, such as Visual Studio Code.
+
+   The `<h1>` element contains the new text, "I Love Cake", because you made the change using the **Sources** tool's editor to edit `index.html` and then saved the change, and that file was mapped in a workspace (the **Workspace** tab), indicated by a green dot on the file's icon.
+
+Continue with the next section.
 
 
 <!-- ====================================================================== -->
-## Edit JavaScript and save changes to the source file
+## Step 4: Save a JavaScript change to disk
+<!-- https://developer.chrome.com/docs/devtools/workspaces/#js -->
+<!--
+subsection order for Edge doc:
+1. Open the Quick source tool without using Command Menu.
+2. Open the Quick source tool by using Command Menu.
+3. Open a .js file in the Quick source tool without using Command Menu.
+4. Open a .js file in the Quick source tool by using Command Menu.
+-->
+
+Next, edit JavaScript and save changes to the source file.
 
 The main place to use the code editor of DevTools is the **Sources** tool.  But sometimes you need to access other tools, such as the **Elements** tool or the **Console**, while editing files.  The **Quick source** tool gives you just the editor from the **Sources** tool, while any tool is open.
-<!-- todo: avoid Quick source tool and just use Sources tool for this section -->
 
-To open the DevTools code editor alongside other tools:
+There are a couple ways to open the **Quick source** tool, and a couple ways to open a file in the **Quick source** tool.  We'll walk through the regular, GUI way and then the Command Menu shortcut way.
```

Comment:
I'm not convinced that we should talk about the Quick source tool here. We have other articles that mention Quick source and explain how to open it and make it display files. This article is all about the Workspace tab. The Workspace tab is already really complicated as it is. I'm not sure we're helping much by telling people that they can also change code in the Quick source tool.

I think we should focus this article on the main scenario for using Workspace, which is to set it up and then use it in the Sources tool.
And then, if people really want to view and edit source files at the same time as using another tool, they can go and learn about it in the article that's about Quick source.

Reply:
I was wondering/ doubting the same thing: seems like a jump of topic, to try to make Workspace tut teach instead about Quick source tool. @captainbrosset

Reply:
removed QS, redid png